src/common/bitfield.h

```diff
@@ -86,13 +90,15 @@ struct BitFieldContainer {
 
  public:
   BitFieldContainer() = default;
-  XGBOOST_DEVICE explicit BitFieldContainer(common::Span<value_type> bits) : bits_{bits} {}
-  XGBOOST_DEVICE BitFieldContainer(BitFieldContainer const& other) : bits_{other.bits_} {}
```

Comment:
Is it your intention that we shouldn't allow copy-construction of `BitFieldContainer` on device?

Reply:
When `default` is specified, nvcc will treat it as host-device function automatically.